events/bingo/bingoBossApp/ui/bingoBossApp_ui.html

```diff
@@ -0,0 +1,35 @@
+<html>
+    <head>
+        <title>Bingo</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="https://fonts.googleapis.com/css?family=Raleway:300,400,600,700|Inconsolata" rel="stylesheet">
+        <link rel="stylesheet" href="css/bingoBossApp_ui.css?2">
```

Comment:
What’s your take on leaving ?## In the production code?

Reply:
necessary sometimes (for testing), doesn't hurt anything when it's present and it's not necessary